CHANGELOG.md

```diff
@@ -1,24 +1,1031 @@
+
+See `ipfs diag profile --help` for more details.
+
+For general debugging information, see [the debug guide](https://github.com/ipfs/go-ipfs/blob/master/docs/debug-guide.md).
+
+#### 🔑 Support for PEM/PKCS8 for key import/export
+
+It is now possible to import or export private keys wrapped in interoperable [PEM PKCS8](https://en.wikipedia.org/wiki/PKCS_8) by passing `--format=pem-pkcs8-cleartext` to `ipfs key import` and `export` commands.
+
+This improved interop allows for key generation outside of the IPFS node:
+
+```console
+$ openssl genpkey -algorithm ED25519 > ed25519.pem
+$ ipfs key import test-openssl -f pem-pkcs8-cleartext ed25519.pem
+```
+
+Or using external tools like the standard `openssl` to get a PEM file with the public key:
+```console
+  $ ipfs key export testkey --format=pem-pkcs8-cleartext -o privkey.pem
+  $ openssl pkey -in privkey.pem -pubout > pubkey.pem
+```
+
+#### 🧹 Using standard IPLD codec names across the CLI/HTTP API
+
+This release makes necessary (breaking) changes in effort to use canonical codec names from [multicodec/table.csv](https://github.com/multiformats/multicodec/blob/master/table.csv). We also switched to CIDv1 in `block put`.  The breaking changes are discussed above.
+
+
+#### 🐳 Custom initialization for Docker
+
+Docker images published at https://hub.docker.com/r/ipfs/go-ipfs/  now support custom initialization by mounting scripts in the `/container-init.d` directory in the container. Scripts can set custom configuration using `ipfs config`, or otherwise customize the container before the daemon is started.
+
+Scripts are executed sequentially and in lexicographic order, before the IPFS daemon is started and after `ipfs init` is run and the swarm keys are copied (if the IPFS repo needs initialization).
+
+For more information, see:
+- Documentation: [ Run IPFS inside Docker](https://docs.ipfs.io/how-to/run-ipfs-inside-docker/) <!-- TODO: needs https://github.com/ipfs/ipfs-docs/pull/1115/files -->
+- Examples in [ipfs-shipyard/go-ipfs-docker-examples](https://github.com/ipfs-shipyard/go-ipfs-docker-examples).
+
+#### RPC API docs for experimental and deprecated commands
+
+https://docs.ipfs.io/reference/http/api/ now includes separate sections for _experimental_ and _deprecated_ commands.
+
+We also display a warning in the command line:
+
+```console
+$ ipfs name pubsub state --help
+WARNING:   EXPERIMENTAL, command may change in future releases
+```
+
+### Art Interlude
+
+[![](https://upload.wikimedia.org/wikipedia/commons/thumb/7/75/Anonymous_Flemish_master_-_Young_woman_on_her_deathbed.jpg/800px-Anonymous_Flemish_master_-_Young_woman_on_her_deathbed.jpg?20150401134921)](https://commons.wikimedia.org/wiki/File:Anonymous_Flemish_master_-_Young_woman_on_her_deathbed.jpg)
+_IPFS Steward after finishing writing go-ipfs Release Notes,_ oil on canvas, circa 1621
```

Comment:
As true as this, let's remove this. Thanks @lidel for all the hard work here 🙏 ❤️